tests/open_alchemy/test_build.py

```diff
@@ -585,3 +603,69 @@ def test_execute(tmp_path):
     assert "class TSchema" in init_contents
     assert "id: 'sqlalchemy.Column[typing.Optional[int]]'" in init_contents
     assert "Schema: typing.Type[TSchema]" in init_contents
+
+    # Assert one package per requested format is created.
+    for extension in extensions:
+        dist_dir = project_path / "dist"
+        files = list(dist_dir.glob(f"{name}*{extension}"))
+        assert len(files) == 1
+
+
+@pytest.mark.integration
+def test_build_dist_wheel_import_error(tmp_path):
+    """
+    GIVEN wheel package is not available
+    WHEN build_dist is called
+    THEN RuntimeError is raised.
+    """
+    dist = tmp_path / "dist"
+    dist.mkdir()
+
+    name = "app_models"
+    version = "version 1"
+    spec = {
+        "info": {
+            "version": version,
+        },
+        "components": {
+            "schemas": {
+                "Schema": {
+                    "type": "object",
+                    "x-tablename": "schema",
+                    "properties": {"id": {"type": "integer"}},
+                }
+            }
+        },
+    }
+
+    try:
+        build.run("pip uninstall -y wheel", ".")
+        with pytest.raises(RuntimeError):
+            build.execute(
+                spec=spec, name=name, path=str(dist), format_=PackageFormat.WHEEL
+            )
+    finally:
+        build.run("pip install wheel", ".")
+
+
+@pytest.mark.build
+def test_validate_dist_format_valid():
+    """
+    GIVEN a valid format
+    WHEN validate_dist_format is called
+    THEN nothing happens
+    """
+    for format_ in PackageFormat:
```

Comment:
My preference would be to have this be parametrised. What about the case where both `wheel` and `sdist` are given?

Reply:
Addressed ✅